samples/getting-started/android/app/build.gradle

```diff
@@ -19,8 +16,20 @@ android {
             minifyEnabled false
         }
     }
-    externalNativeBuild.cmake.path "CMakeLists.txt"
-
+    splits {
+        abi {
+            reset()
+            enable true
+            universalApk false
+            include "armeabi-v7a"
+        }
+    }
+    externalNativeBuild {
+        cmake {
+            version "3.16.2"
+            path "CMakeLists.txt"
```

Comment:
not sure we need this?

Reply:
pretty sure, i removed path first and then the entire block. In both cases gradle builds the app without any native stuff